launcher

```diff
@@ -224,6 +225,25 @@ restart() {
     start
 }
 
+gc() {
+    ensure_container_running
+    (
+        loginfo "Stop seafile and start garbage collector"
+        docker exec -it seafile /bin/bash -c "\
+            /opt/seafile/seafile-server-latest/seafile.sh stop; \
+            sleep 10; \
+            /opt/seafile/seafile-server-latest/seaf-gc.sh >> /var/log/gc.log; \
```

Comment:
Use `tee` instead of simple redirection. It would be useful to show the output of the gc command in real time.